packages/story-editor/tsconfig.json

```diff
@@ -25,6 +25,10 @@
     "src/app/config",
     "src/app/currentUser",
     "src/app/helpCenter",
+    "src/components/library/paneIds.ts",
+    "src/components/sidebar/constants.ts",
+    "src/app/highlights/*.ts",
+    "src/app/highlights/provider.tsx",
```

Comment:
Just `src/app/highlights` should work, no? Or are there some files you're not converting?

Reply:
No style.js is not TS.